File: hotel_search/v2/get/hotel_offers/Node SDK/hotel_search.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// Get list of Hotels by city code
amadeus.shopping.hotelOffers.get({
  cityCode: 'MAD'
}).then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});

File: hotel_search/v2/get/hotel_offers/iOS SDK/HotelSearch.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// Get list of hotels by city code
amadeus.shopping.hotelOffers.get(params: ["cityCode": "PAR"],
    onCompletion: { result in
        switch result {
        case let .success(response):
            print(response.data)
        case let .failure(error):
            fatalError(error.localizedDescription)
        }
    })
